scripts/pre-gen/generateOpenapitools.ts

```diff
@@ -25,10 +25,12 @@ export async function generateOpenapitools(gens: Generator[]): Promise<void> {
       gitRepoId: getClientsConfigField(language, 'gitRepoId'),
       glob: `specs/bundled/${client}.yml`,
       templateDir,
-      generatorName: `algolia-${language}`,
-      output: `#{cwd}/${rest.output}`,
+      generatorName: `algolia-${mode !== 'client' ? 'cts' : language}`,
+      output: `#{cwd}/${mode === 'client' ? rest.output : ''}`,
```

Comment:
same ternary operator maybe just for consistency

```suggestion
      generatorName: `algolia-${mode === 'client' ? language : 'cts'}`,
      output: `#{cwd}/${mode === 'client' ? rest.output : ''}`,
```